istioctl/pkg/validate/validate.go

```diff
@@ -158,6 +163,39 @@ func (v *validator) validateResource(istioNamespace string, un *unstructured.Uns
 		v.validateDeploymentLabel(istioNamespace, un)
 		return nil
 	}
+
+	if un.GetAPIVersion() == "install.istio.io/v1alpha1" {
```

Comment:
is the GVK a constant defined somewhere we can reuse?

Reply:
No.  This work would have been easier if IstioOperator was like the other other Istio Resources and had a Golang "schema" and deserialized using the same logic as the other Istio Unstructureds.  (I am not sure where to propose that kind of factoring.)